### DISCUSSION TITLE
2.4.0

### DISCUSSION THREAD

**Discussion by cschramm** [blueman-project/blueman#2179] [General]
I think it's time for a 2.4 release. Revisiting https://github.com/blueman-project/blueman/milestone/13 I tend to just postpone all the signal stuff (and to drop it altogether in 2.5 🫣).

What is probably worth looking into is the cluster of #2163 #2161 #1119 #1707.

Anything else we need to add (except translations, of course)?

**Comment by cschramm** [comment]
It's been far to long. Anything to add to a 2.4.0, @infirit?

**Reply by cschramm** [reply]
I'm finally going to cut 2.4.0 from the current state, plus any translations that make it to #2245 before I do. Are you fine with that, @infirit?

**Comment by infirit** [comment]
Go for it. But I just realized I haven't updated the changelog for my recent change. I'll go over the git log and update it tomorrow.
